data/mods/CBM_Dungeon_Test/labyrinth_monsters.json

```diff
@@ -0,0 +1,157 @@
+[
+  {
+    "id": "mon_fleshborg_1_1",
+    "//": "Labyrinth monster naming convention: first number - amount of 'chaos' (How weird is this creature).  Second number - difficulty level.",
+    "type": "MONSTER",
+    "name": { "str": "fleshborg" },
+    "description": "Compared to the cyborgs you've seen elsewhere, this one looks almost inverted.  Its body is made of smooth, segmented, paper-thin metallic plates; fleshy protrusions and bits of bone jut from components growing off this base frame.  It moves wildly, but not like a zombie; more like a machine gone haywire.",
+    "default_faction": "nether_hate_humans",
+    "looks_like": "mon_zomborg",
+    "volume": "120 L",
+    "weight": "200 kg",
+    "species": [ "CYBORG", "ROBOT" ],
+    "bodytype": "human",
+    "material": [ "steel", "flesh" ],
+    "speed": 80,
+    "symbol": "B",
+    "morale": 100,
+    "aggression": 100,
+    "hp": 85,
+    "color": "light_gray",
+    "melee_skill": 5,
+    "dodge": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "cut", "amount": 12 } ],
+    "bleed_rate": 25,
+    "vision_day": 15,
+    "vision_night": 10,
+    "starting_ammo": { "bone_dart": 10 },
+    "special_attacks": [
+      {
+        "type": "gun",
+        "cooldown": 5,
+        "move_cost": 100,
+        "gun_type": "feral_human_thrown_rock",
+        "ammo_type": "bone_dart",
+        "fake_skills": [ [ "throw", 4 ] ],
+        "fake_str": 10,
+        "condition": { "not": { "u_has_effect": "maimed_arm" } },
+        "require_targeting_player": false,
+        "ranges": [ [ 2, 5, "DEFAULT" ] ],
+        "description": "The fleshborg launches a bony dart at you!"
+      }
+    ],
+    "death_drops": { "subtype": "collection", "groups": [ [ "robots", 80 ] ] },
+    "weakpoint_sets": [ "wps_cyborg_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology", "prof_wp_cyborg" ],
+    "harvest": "mon_zomborg",
+    "dissect": "dissect_mon_zomborg",
+    "armor": { "bash": 6, "cut": 28, "bullet": 16 },
+    "flags": [ "SEES", "HEARS", "WARM", "BASHES", "GROUP_BASH", "PATH_AVOID_DANGER_1", "PUSH_MON" ]
+  },
+  {
+    "id": "mon_fleshborg_1_2",
+    "//": "Labyrinth monster naming convention: first number - amount of 'chaos' (How weird is this creature).  Second number - difficulty level.",
+    "type": "MONSTER",
+    "name": { "str": "advanced fleshborg" },
+    "description": "Compared to the cyborgs you've seen elsewhere, this one looks almost inverted.  Its body is made of smooth, segmented, paper-thin metallic plates; fleshy protrusions and bits of bone jut from components growing off this base frame.  It moves with dire purpose, like a trained combat machine.",
+    "copy-from": "mon_fleshborg_1_1",
+    "melee_skill": 6,
```

Comment:
Might be worth considering using relative/proportional for more difficult versions to ensure they're more difficult (at least numerically)
Presuming these fields support it anyway I haven't touched monsters much

Reply:
Yes, you're probably right. I don't think these are going to survive more than a few passes as is but I'll see about switching over when I adjust them next.